app/assets/stylesheets/components/_buttons.scss

```diff
@@ -10,10 +9,12 @@
     margin: 0 10px 20px 30px;
     outline: none;
     width: 80%;
+    @include themify {

```

Comment:
Is this the best way to do it??
It seems youre reaptigin the code too much

Reply:
And update the branch. that button should be there you should constantly be doing pulls from main

Reply:
Done